src/main/java/ch/njol/skript/expressions/ExprEntities.java

```diff
@@ -180,6 +195,30 @@ public Iterator<? extends Entity> iterator(Event e) {
 					}
 					return false;
 				});
+		} else if (isUsingCuboid) {
+			assert from != null;
+			Location corner1 = from.getSingle(e);
+			if (corner1 == null) {
+				return null;
+			}
+			assert to != null;
+			Location corner2 = to.getSingle(e);
+			if (corner2 == null) {
+				return null;
+			}
```

Comment:
```suggestion
			if (corner2 == null)
				return null;
```
optional but i think it looks much better
goes for the others too